app/managers.py

```diff
@@ -0,0 +1,41 @@
+import sqlite3
+from app.models import Actor
+
+
+# from models import Actor
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
```

Comment:
Consider adding functionality to create the database and the 'actors' table if they do not exist. This will ensure that the application can run independently without requiring manual setup of the database.